k2/docs/source/_k2.py

```diff
@@ -0,0 +1,17 @@
+# this file is for generating documentation
+
```

Comment:
So do the inclusion of these things make it document them?   Or not document them?

Reply:
No, the purpose is to make `import k2` working as `k2` depends on `_k2`. The code is not going to be executed.
Everything that is going to be generated is specified explicitly in
https://github.com/k2-fsa/k2/blob/master/k2/docs/source/python_api/index.rst

```rst
Python API Reference
====================

.. toctree::
   :maxdepth: 2
   :caption: Python API

   fsa
   fsa_algo
   symbol_table
```

If we want to document some of the functions from `_k2`, we can document them here, add a file `python_api/_k2.rst`, and add it to `python_api/index.rst`.